rootfs/api/models/config.py

```diff
@@ -137,13 +111,12 @@ def save(self, **kwargs):
                         # error if unsetting non-existing key
                         if key not in data:
                             raise UnprocessableEntity('{} does not exist under {}'.format(key, attr))  # noqa
-
                         data.pop(key)
                     else:
                         data[key] = value
                 setattr(self, attr, data)
 
-            self.set_healthchecks()
+            self._migrate_legacy_healthcheck()
```

Comment:
The reason I din't migrate before I left is how are we gonna handle port.... current environment health checks doesn't provide port and we use to get it from releases in the scheduler logic.... Port is the only mandatory field and we won't have handle of it in the config object